frontend/src/containers/Profile/ProfileDetail/ProfileDetail.js

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from "react";
+import {
+    Button, Image, Tabs, Tab,
+} from "react-bootstrap";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+// import { connect } from "react-redux";
+
+import { SideBar, Header } from "../../../components";
+import CollectionCard from "../../../components/Collection/CollectionCard/CollectionCard";
+import ReviewCard from "../../../components/Review/ReviewCard/ReviewCard";
+
+import "./ProfileDetail.css";
+import SamplePhoto from "./sample.jpg";
+// import * as actionCreators from "../../../store/actions/index";
+
+class ProfileDetail extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            doIFollow: this.props.thisUser.doIFollow,
+        };
+    }
+
+    render() {
+        const editButton = (
+            <Link to={`/profile/${this.props.thisUser.id}/edit`}>
+                <Button id="editButton">Edit</Button>
+            </Link>
+        );
+        const followButton = <Button id="followButton" onClick={() => this.setState({ doIFollow: true })}>Follow</Button>;
+        const unfollowButton = <Button id="unfollowButton" onClick={() => this.setState({ doIFollow: false })}>Unfollow</Button>;
+        // only one button will be displayed among "edit", "follow", and "unfollow" buttons
+        let buttonDisplayed;
+        if (this.props.currentUserID === this.props.thisUser.id) {
+            buttonDisplayed = editButton;
+        } else if (this.state.doIFollow) {
+            buttonDisplayed = unfollowButton;
+        } else {
+            buttonDisplayed = followButton;
+        }
+
+        const collectionCardsLeft = this.props.thisUserCollections
+            .filter((x) => this.props.thisUserCollections.indexOf(x) % 2 === 0)
+            .map((collection) => (
+                <CollectionCard
+                  key={collection.id}
+                  source={collection.source}
+                  id={collection.id}
+                  user={collection.user}
+                  title={collection.title}
+                  numPapers={collection.numPapers}
```

Comment:
We should change 'num~' to '~Count'.(ex> likeCount) You can refer to Main Page for it.

Reply:
Please pull the latest master branch! They were changed in recent versions.